packages/cli/src/commands/trace.ts

```diff
@@ -74,7 +76,12 @@ const commandModule: CommandModule<Options, Options> = {
     for (const name of names) {
       const systemSelector = new TableId(namespace, name);
       // Get the first field of `Systems` table (the table maps system name to its address and other data)
-      const address = await WorldContract.getField(systemsTableId.toHex(), [systemSelector.toHex()], 0);
+      const address = await WorldContract.getField(
+        systemsTableId.toHex(),
+        [systemSelector.toHex()],
+        0,
+        systemTableSchema
+      );
```

Comment:
`getField` now requires the value schema to be passed in

Reply:
another approach that might be a little less error prone if we change this schema later and forget to update this is to fetch the schema from the chain before we get field